regions/io/core.py

```diff
@@ -188,16 +190,22 @@ class Shape(object):
     ----------
     format_type : str
         File Format type
+
```

Comment:
there is not supposed to be whitespace between these entries

Reply:
ok. will fix it, but it looks better this way, hehe!